projects/core/src/product/store/reducers/product-reviews.reducer.spec.ts

```diff
@@ -1,5 +1,6 @@
 import * as fromReducer from './../reducers/product-reviews.reducer';
```

Comment:
Can't comment there. On the non test version of this file please add this line 30: 
```
export const getReviewList = (state: ProductReviewsState): Review[] =>
  state.list;
```

This is a fix for an issue with Typescript version 2.9